### ISSUE TITLE
Usability Concern: Context menu item "Show in File Manager" has non-standard naming on Mac/Windows

### ISSUE THREAD

**Issue by mattlubner** [atom/tree-view#1307]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Opening a new issue here at the request of @50Wliu (original issue: atom/atom#18694).

The menu item "Show in File Manager" previously followed the naming conventions of the user's operating system (either "Show in Finder", "Show in Explorer", or "Show in File Manager", for Mac, Windows, or Linux, respectively). It's now hard-coded to "Show in File Manager". This is a usability regression that was accepted as part of #1266 (because it's no longer possible to know exactly which file manager will be used).

I'd like to propose reverting the name-change portion of #1266 (except on Mac it should be named "Reveal in Finder"), so that the operating system's naming conventions are followed.

I believe fewer users will be confused by pairing the old-but-accepted terminology with Atom opening the file or folder in their chosen third-party file manager, than are confused by the non-standard term "Show in File Manager" (which sounds like it invokes some new / custom Atom functionality). It stands to reason that more users expect the standard terminology than use a third-party file manager.

Would anyone have an objection to this change?

I'd be happy to submit a PR of exactly what I mean, if that would be helpful.

### Versions

```
macOS Mojave 10.14.3 (18D109)
```

```sh
# atom --version
Atom    : 1.35.1
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

**Comment by rsese** [comment]
Thanks for opening a separate issue for this @mattlubner -

> I'd like to propose reverting the name-change portion of #1266 (except on Mac it should be named "Reveal in Finder"), so that the operating system's naming conventions are followed.

...

> I'd be happy to submit a PR of exactly what I mean, if that would be helpful.

We talked about this and are :+1: on this change, thanks for checking with us :bow:

**Comment by 50Wliu** [comment]
This has been fixed in #1311.
